modules/effects/arrow-effects-data/src/test/kotlin/arrow/effects/IOTest.kt

```diff
@@ -232,7 +232,7 @@ class IOTest : UnitSpec() {
       val order = mutableListOf<Long>()
 
       fun makePar(num: Long) =
-        IO(newSingleThreadContext("$num")) {
+        IO(newCoroutineDispatcher("$num")) {
```

Comment:
This is certainly not correct because `newCoroutineDispatcher` underlying uses a `ThreadPoolExecutor` with a corePoolSize = 10 and a maxPoolSize = 50. We're using `newSingleThreadContext` to test that parallel code still works correcrtly when running on a single thread.